docs/tutorials/text_prediction/automm.md

```diff
@@ -153,241 +95,211 @@ train_data = train_data.sample(500, random_state=0)
 
+The `data` config defines some model-agnostic rules in preprocessing data. Note that `AutoMMPredictor` converts categorical data into text by default.
 
-```{.python .input}
-predictor.save('my_saved_dir')
-loaded_predictor = AutoMMPredictor.load('my_saved_dir')
-scores2 = loaded_predictor.evaluate(test_data, metrics=["accuracy"])
-print(scores2)
-```
+The `optimization` config has hyper-parameters for model training. `AutoMMPredictor` uses layer-wise learning rate decay, which decreases the learning rate gradually from the output to the input end of one model.
 
-This quick demo probably results in low scores due to the tiny backbone, subsampled data, and time limit. You may try a larger backbone and avoid setting the time limit in practice.
+The `env` config contains the environment/machine related hyper-parameters. For example, the optimal values of `per_gpu_batch_size` and `per_gpu_batch_size_evaluation` are closely related to the GPU memory size.
 
-## Multimodal Prediction
-One significant merit of AutoMM is its scalability to multiple modalities, including image, text, numerical, and categorical data. Given one multimodal dataframe, AutoMM can automatically detect the modality of each column and then select the suitable backbones for different modalities. If more than one individual backbones are available, it would add a MLP fusion module on the top to fuse their output features to make the final prediction. Thanks to the flexible design, it's easy to apply AutoMM to any image, text, numerical, and categorical data combination. AutoMM can also infer the problem type (classification or regression) by analyzing the label column.
+You can flexibly customize any hyper-parameter in `config` via the `hyperparameters` argument of `.fit()`. To access one hyper-parameter in `config`, you need to traverse from top-level keys to bottom-level keys and join them together with `.` For example, if you want to change the per GPU batch size to 16, you can set `hyperparameters={"env.per_gpu_batch_size": 16}`.
 
-### TIMM + Huggingface Transformers + More
-AutoMM maximizes the choices of image and text backbones by supporting [timm](https://github.com/rwightman/pytorch-image-models) and [huggingface transformers](https://github.com/huggingface/transformers). AutoMM uses MLP for numerical data but converts categorical data to text by default. 
+## APIs
+Besides `.fit()` and `.evaluate()`, `AutoMMPredictor` also provides other useful APIs.
```

Comment:
It's not a bad idea to mention that it adopts a similar API as TextPredictor and TabularPredictor. You may use. 

```
Besides `.fit()` and `.evaluate()`, `AutoMMPredictor` also provides other useful APIs that are similar to that in TextPredictor and TabularPredictor. You may refer to more details in `:ref:sec_textprediction_beginner`.
```

Reply:
Sure.